browser/brave_browser_process_impl.h

```diff
@@ -155,6 +151,7 @@ class BraveBrowserProcessImpl : public BraveBrowserProcess,
 #endif
   std::unique_ptr<debounce::DebounceComponentInstaller>
       debounce_component_installer_;
+  bool created_https_everywhere_service_ = false;
```

Comment:
not sure why this is needed

Reply:
as explained above this was changed to match upstream conventions